.ci/azure/linux.yml

```diff
@@ -114,6 +118,19 @@ jobs:
       # 'clang' compiler is to check that samples can be built using it
       sudo apt --assume-yes install openjdk-11-jdk clang
       # For opencv-python: python3-setuptools and pip upgrade
+      wget https://www.python.org/ftp/python/"$PYTHON_VERSION"/Python-"$PYTHON_VERSION".tgz
+      tar -xzf Python-"$PYTHON_VERSION".tgz
+      cd Python-"$PYTHON_VERSION" || exit
+      ./configure --enable-optimizations --prefix=/usr/local --enable-shared LDFLAGS="-Wl,-rpath /usr/local/lib"
+      make -j "$NUM_PROC" python
+      sudo make -j "$NUM_PROC" install
+      export LD_LIBRARY_PATH=/usr/local/lib:$LD_LIBRARY_PATH
+      /usr/local/bin/python3.10 -m venv $(PY_VENV)
+      . $(PY_VENV)/bin/activate
+      echo Python3 info ; which python3 ; python3 --version
+      python3 --version
+      python3.10 --version
+      curl https://bootstrap.pypa.io/get-pip.py | python3
```

Comment:
why do we need all of this? can we just switch azure on U22? where python 3.10 is default